multicast/recv.py

```diff
@@ -270,8 +274,7 @@ def hearstep(groups, port, iface=None, bind_group=None):
 			if chunk is not None:
 				msgbuffer += str(chunk, encoding='utf8')
 				chunk = None
-				msgbuffer += unicodedata.lookup("""SOFT HYPHEN""")
-			#print(sock.recv(1316))
+				# msgbuffer += unicodedata.lookup("""SOFT HYPHEN""")
 			# about 969 bytes in base64 encoded as chars
 	except KeyboardInterrupt:
 		print("""""")
```

Comment:
```suggestion
		print(str(""""""))
```

***OR***
add:
```python
__SPACER_CONSTANT = str("""""")
```

```suggestion
		print(__SPACER_CONSTANT)
```

Reply:
:no_good: code looks worse with longer variable name and is harder to read with shorter less descriptive variable names

```suggestion
		print(__BLANK)
```
might work though

- [ ] Opened a new issue for this

Reply:
:tada: added #21 for this